tic_tac_toe/game.py

```diff
@@ -9,7 +9,13 @@ def _position_is_empty_in_board(position, board):
 
     Returns True if given position is empty, False otherwise.
     """
-    pass
+    #tuple = (1,0)
+    x = position[0]
+    y = position[1]
+    if board[x][y] != '-':
+        return False
```

Comment:
Wondering why you used "!=" and False instead of "==" and True? Avoiding 'not' when possible might improve readability.